app/assets/javascripts/components/cdx_select.js.jsx

```diff
@@ -16,21 +16,27 @@ var CdxSelect = React.createClass({
     }
   },
 
+  onInputChange: function (newValue) {
+    this.selectRef.setValue(newValue)
```

Comment:
I'd suggest to check for `this.props.searchable` in this callback, and to add a comment explaining why we're updating the selectRef value or not.

Reply:
do you mean something like this? 

```  
 if(this.props.searchable){
   this.selectRef.setValue(newValue) 
 }
```

Reply:
we already check for that property here: `onInputChange={this.props.searchable && this.onInputChange}>`, adding that check would redundant, wouldn't it? Or there is something that I'm missing?